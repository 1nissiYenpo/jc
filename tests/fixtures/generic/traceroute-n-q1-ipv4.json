{"destination_ip":"199.58.80.40","destination_name":"www.koumbit.org","hops":[{"hop":1,"probes":[{"annotation":null,"asn":null,"ip":"192.168.2.1","name":null,"rtt":3.425}]},{"hop":2,"probes":[{"annotation":null,"asn":null,"ip":"24.212.242.17","name":null,"rtt":16.153}]},{"hop":3,"probes":[{"annotation":null,"asn":null,"ip":"10.170.192.58","name":null,"rtt":17.231}]},{"hop":4,"probes":[{"annotation":null,"asn":null,"ip":"192.171.61.161","name":null,"rtt":25.393}]},{"hop":5,"probes":[{"annotation":null,"asn":null,"ip":"206.248.189.97","name":null,"rtt":25.322}]},{"hop":6,"probes":[{"annotation":null,"asn":null,"ip":"198.179.18.41","name":null,"rtt":23.755}]},{"hop":7,"probes":[{"annotation":null,"asn":null,"ip":"64.15.69.54","name":null,"rtt":25.091}]},{"hop":8,"probes":[{"annotation":null,"asn":null,"ip":"199.58.80.40","name":null,"rtt":25.196}]}]}
