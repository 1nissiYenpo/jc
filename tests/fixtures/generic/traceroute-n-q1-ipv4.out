traceroute to www.koumbit.org (199.58.80.40), 30 hops max, 60 byte packets
 1  192.168.2.1  3.425 ms
 2  24.212.242.17  16.153 ms
 3  10.170.192.58  17.231 ms
 4  192.171.61.161  25.393 ms
 5  206.248.189.97  25.322 ms
 6  198.179.18.41  23.755 ms
 7  64.15.69.54  25.091 ms
 8  199.58.80.40  25.196 ms
